ambry-api/src/main/java/com/github/ambry/config/CompositeAccountServiceConfig.java

```diff
@@ -48,12 +52,28 @@
   @Default("50")
   public final int samplingPercentageForGetConsistencyCheck;
 
+  /**
+   * The AccountServiceFactory that is used in {@code CompositeAccountService} as primary source of account-related information.
+   */
+  @Config(PRIMARY_ACCOUNT_SERVICE_FACTORY)
+  @Default("com.github.ambry.account.HelixAccountServiceFactory")
+  public final String primaryAccountServiceFactory;
+
+  /**
+   * The AccountServiceFactory that is used in {@code CompositeAccountService} as secondary source of account-related information.
+   */
+  @Config(SECONDARY_ACCOUNT_SERVICE_FACTORY)
+  @Default("com.github.ambry.account.MySqlAccountServiceFactory")
+  public final String secondaryAccountServiceFactory;
+
   public CompositeAccountServiceConfig(VerifiableProperties verifiableProperties) {
     consistencyCheckerIntervalMinutes =
         verifiableProperties.getIntInRange(CONSISTENCY_CHECKER_INTERVAL_MINUTES, 5, 0, Integer.MAX_VALUE);
     consistencyCheckerShutdownTimeoutMinutes =
         verifiableProperties.getIntInRange(CONSISTENCY_CHECKER_SHUTDOWN_TIMEOUT_MINUTES, 1, 0, Integer.MAX_VALUE);
     samplingPercentageForGetConsistencyCheck =
         verifiableProperties.getIntInRange(SAMPLING_PERCENTAGE_FOR_GET_CONSISTENCY_CHECK, 50, 0, 100);
+    primaryAccountServiceFactory = verifiableProperties.getString(PRIMARY_ACCOUNT_SERVICE_FACTORY);
```

Comment:
Need to either specify default here or add the properties in CompositeAccountServiceTest.